tests/test_quicksetup.py

```diff
@@ -0,0 +1,16 @@
+import unittest
+import bpy
+
+class TestQuickSetup(unittest.TestCase):
+    def test_emission(self):
+        bpy.ops.node.y_quick_ypaint_node_setup(type='EMISSION')
+        
+        nodes = bpy.data.materials['Material'].node_tree.nodes
+
+        ucupaint = False
```

Comment:
Do not use a singular vague variable like this, also `ucupaint` is not a valid name internally, it should be called `yp` instead. So the correct variable will be `is_yp_node_found`